lib/main_common.pm

```diff
@@ -594,4 +595,34 @@ sub load_iso_in_external_tests {
     loadtest "x11/reboot_and_install";
 }
 
+sub load_x11regression_documentation {
+    if (check_var("DESKTOP", "gnome")) {
```

Comment:
I would prefer this to be an early-return, i.e.

```perl
return unless check_var('DESKTOP', 'gnome');
```

to reduce complexity

Reply:
I will update this in my next commit.